src/main/scala/lms/transformation/mpi_nccl_transformation/distributedTensor2MPINCCLBase.scala

```diff
@@ -54,6 +54,12 @@ abstract class DistributeTensor2MPI_NCCLBase extends Transformer with MPIOps wit
     CUDA_SET_DEVICE(device)
     CUDA_MALLOC(size, m)
   }
+
+  // TODO: change it to better name, and add INT counterparts to other gpu functions as well
+  def gpu_array1_bytes(size: INT, m: Manifest[_], device: INT)(implicit __pos: SourceContext): ARRAY = {
```

Comment:
would `gpu_array_by_byte` be a good name?

Reply:
yeah, that's more descriptive.
btw I think we should also change the `array1` name. I think this fix can be made after the current branched are merged.

Reply:
fixed in eee74d5